package.json

```diff
@@ -14,6 +14,8 @@
   "author": "OpenZipkin <openzipkin.alt@gmail.com>",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
```

Comment:
Another option would be to just run the whole code base through babel to make it node 4-compatible, and then just set up the build process etc. to use node 4. So we don't need a build step.